md/workspaces-and-repositories.md

```diff
@@ -101,32 +102,32 @@ Git commands available in the studio interface:
 
  * **Share with Git**  
 This action connects the current project to Git and shares it on a remote.
-To configure the remote, see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-project-on-github.md).
+To configure the remote, see the following [Egit user guide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
```

Comment:
I do not agree with the renaming to repository

Reply:
The file is currently named that way. Link is currently broken.

Reply:
ok then...